packages/odyssey-react/src/components/Field/Field.module.scss

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  display: flex;
+  flex-direction: column;
+  max-width: $max-line-length;
+  margin-block-end: $spacing-m;
+
+  &:last-child {
+    margin-block-end: 0;
+  }
+}
+
+.label {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  width: 100%;
+  margin-block-start: 0;
+  margin-block-end: $spacing-xs;
+  margin-inline: 0;
+  color: $text-heading;
+  font-weight: 600;
+}
+
+.optionalLabel {
+  padding-inline-start: $spacing-s;
+  color: $text-sub;
+  font-size: $size-body-caption;
+  font-weight: 400;
+  line-height: $spacing-m; // Helps optically match line-height above
+}
+
+.hint {
+  margin-block-start: -$spacing-xs; // Hug label
```

Comment:
Got it - we can leave this as is for now, but keep in mind that even component groupings that are all assigned as statics to the same base like `Form.Field` or `Form.FieldGroup` can have contextual styles. In general we want to work to avoid these types of CSS solutions because they are brittle.

A more verbose, but explicit way to solve this is to have two classes applied to label, one that is applied as a base and contains sensible defaults for margin, and a second that is applied via JS only when a hint is rendered alongside so that the margin is updated correctly on label itself without having to use negative margin on an adjacent hint element cc @arnoldsandoval-okta @andrewberg-okta